docs/csharp/quick-starts/interpolated-strings.yml

```diff
@@ -0,0 +1,176 @@
+   
+       public override string ToString() => Name;
+    }
+
+    public class Example
+    {
+       public enum Unit { item, pound, ounce, dozen };
+   
+       public static void Main()
+       {
+          var item = new Vegetable("eggplant");
+          var date = DateTime.Now;
+          var price = 1.99m;
+          var unit = Unit.item;
+          Console.WriteLine($"On {date:d}, the price of {item} was {price:C2} per {unit}.");
+       }
+    }
+    ```
+     You specify a format string by following the interpolated expression with a colon and the format string. "d" is a [standard date and time format string](../../standard/base-types/standard-date-and-time-format-strings.md#the-short-date-d-format-specifier) that represents the short date format. "C2" is a  [standard numeric format string](../../standard/base-types/standard-numeric-format-strings.md#the-currency-c-format-specifier) that represents a number as a currency value with two digits after the decimal point.
+
+    A number of types in the .NET Standard libraries support a predefined set of format strings. These include all the numeric types and the date and time types. For a complete list of types that support format strings, see [Format Strings and .NET Class Library Types](../../standard/base-types/formatting-types.md#stringRef) in the [Formatting Types in .NET](../../standard/base-types/formatting-types.md) article. Any type can support a set of format strings, and you can also develop custom formatting extensions that provide custom formatting for existing types. For information on custom formatting by providing an <xref:System.ICustomFormatter> implementation, see [Custom Formatting with ICustomFormatter](../../standard/base-types/formatting-types.md#custom-formatting-with-icustomformatter) in the [Formatting Types in .NET](../../standard/base-types/formatting-types.md) article.
+
+    Try modifying the the format strings in the **.NET Editor** to see how they affect the formatting of the date and time and the numeric value. Change the "d" in `{date:t}` to "t" (to display the short time format), "y" (to display the year and month), and "yyyy" (to display the year as a four-digit number). Change the "C2" in `{price:C2}` to "e" (for exponential notation) and "F3" (for a numeric value with three digits after the decimal point).
+
+        > [!NOTE]
+    > This online coding experience is in preview mode. If you encounter problems, please report them [on the dotnet/try repo](https://github.com/dotnet/try/issues).
+
+- title: Control the field width and alignment of interpolated expressions
+  durationInMinutes: 6
+  content: |
+    Ordinarily, when the string returned by an interpolated expression is included in a result string, it has no leading or trailing spaces. Particularly for instances in which you are working with a set of data, interpolated expressions let you specify a field width and its alignment. To see this, run the following code.
+    
+    ```csharp
```

Comment:
This example is really cool, and I'd use it for the accompanying local environment quickstart.  Unfortunately, the width of the interactive output window in the browser make this look unappealing. We should try to find an example that uses fewer columns so the output looks good in this enviornment.

Reply:
Great new example!